pkg/cfg/cfgmgr.go

```diff
@@ -147,16 +149,97 @@ func (c *CfgMgr) getValue(key string) (string, error) {
 	return "", fmt.Errorf("Key value for key %s not found", key)
 }
 
-func (c *CfgMgr) getRandomPassword() string {
-	digits := "0123456789"
-	lowerChars := "abcdefghijklmnopqrstuvwxyz"
-	upperChars := "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
-	allChars := lowerChars + upperChars + digits
+func (c * CfgMgr) getRandomPassword() string {
+    rand.Seed(time.Now().UnixNano())
+    digits := "0123456789"
+    lowerChars := "abcdefghijklmnopqrstuvwxyz"
+    upperChars := "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+    allChars := lowerChars + upperChars + digits
+    const max = 16
+    out := make([]byte, max)
+    for i := 0; i < max; i++ {
+        out[i] = allChars[rand.Intn(len(allChars))]
+    }
+    return string(out)
+}
 
-	const max = 16
-	out := make([]byte, max)
-	for i := 0; i < max; i++ {
-		out[i] = allChars[rand.Intn(len(allChars))]
-	}
-	return string(out)
+func (c *CfgMgr) CreateDB(serviceName, userName string) error {
+    zap.L().Debug("Creating DB for serviceName ", zap.String("serviceName", serviceName))
+    dbserver, err := c.getValue(fmt.Sprintf("%s/keystone/dbserver_key", c.CustomerKeyPrefix))
+    if err != nil {
+    	zap.L().Error("Cannot get dbserver_key from consul store", zap.Error(err))
+        return err
+    }
+    host, err := c.getValue(fmt.Sprintf("%s/host", dbserver))
+    if err != nil {
+    	zap.L().Error("Cannot get host key from consul store", zap.Error(err))
+        return err
+    }
+    port, err := c.getValue(fmt.Sprintf("%s/port", dbserver))
+    if err != nil {
+    	zap.L().Error("Cannot get port key from consul store", zap.Error(err))
+        return err
+    }
+    adminUser, err := c.getValue(fmt.Sprintf("%s/admin_user", dbserver))
+    if err != nil {
+    	zap.L().Error("Cannot get admin_user key from consul store", zap.Error(err))
+        return err
+    }
+    adminPass, err := c.getValue(fmt.Sprintf("%s/admin_pass", dbserver))
+    if err != nil {
+    	zap.L().Error("Cannot get admin_pass key from consul store", zap.Error(err))
+        return err
+    }
+    dbName, err := c.getValue(fmt.Sprintf("%s/%s/db/name", c.CustomerKeyPrefix, serviceName))
+    if err == nil {
+    	zap.L().Info(fmt.Sprintf("Database %s already exists", dbName))
+    	return nil
+    }
+    dbObject, err := sql.Open("mysql", fmt.Sprintf("%s:%s@tcp(%s:%s)/", adminUser, adminPass, host, port))
+    if err != nil {
+        zap.L().Error("Can't connect to MySQL")
+        return err
+    }
+    _, err = dbObject.Exec(fmt.Sprintf("CREATE DATABASE %s", serviceName))
```

Comment:
`mysql> select @@hostname;
+------------------------+
| @@hostname             |
+------------------------+
| mysql-7f4fcc7b87-wqmsl |
+------------------------+`
we capture this value in hostname

Reply:
we are checking if db is present or not at https://github.com/platform9/go-firkinize/pull/1/files#diff-f029d585806a2c597e25fe0422e2db28b6aed18e54b3c1962c8cef5bfb28ec67R193

Reply:
`# ./root/go-firkinize create-db --service-name=hagrid --consul-token=88fc02fe-0034-93a1-6a9e-e56eaedf9cb9 --region-id=5aaaff0a-a48e-491d-8b8d-5c886fe6ae73 --customer-id=15ab505e-ba46-4cd5-a149-65afff82085d --consul-host-port="konsul-dev.platform9.horse:443" --consul-scheme="https"
{"level":"info","ts":1613116062.0443702,"caller":"cfg/cfgmgr.go:195","msg":"Database hagrid already exists"}`
if we run again and if DB already exists, then it will give above message and exit with 0 status